GrandCentralBoard/Widgets/Watch/Server/GoogleCalendarDataProvider.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  GoogleCalendarDataProvider.swift
+//  GrandCentralBoard
+//
+//  Created by Michał Laskowski on 07.04.2016.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import Foundation
+import Result
+
+protocol CalendarDataProvider {
+    func fetchEventsForCalendar(calendarID: String, completion: (Result<[EventModel], APIDataError>) -> Void)
+    func fetchCalendar(calendarID: String, completion: (Result<CalendarModel, APIDataError>) -> Void)
+}
+
+final class GoogleCalendarDataProvider : CalendarDataProvider {
+
+    let dataProvider: APIDataProvider
+
+    init(dataProvider: APIDataProvider) {
+        self.dataProvider = dataProvider
+    }
+
+    private static let dateFormatter: NSDateFormatter =  {
+        let formatter = NSDateFormatter()
+        formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ssZ";
+        return formatter
+    }()
+
+    func fetchEventsForCalendar(calendarID: String, completion: (Result<[EventModel], APIDataError>) -> Void) {
+        guard let escapedCalendarID = calendarID.URLEscape() else {
+            assertionFailure("Failure escaping calendar id: \(calendarID)")
+            return
+        }
+        let url = NSURL(string: "https://www.googleapis.com/calendar/v3/calendars/\(escapedCalendarID)/events")!
```

Comment:
`https://www.googleapis.com/calendar/v3/calendars/`

Define somewhere as `let`. You have identical literal in line `62`.